jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/CachingContentFactory.java

```diff
@@ -285,18 +295,24 @@ public ByteBuffer getBuffer()
 
         public boolean isValid()
         {
-            try
+            if (_delegate.getResource() != null)
             {
-                FileTime lastModifiedTime = Files.getLastModifiedTime(_delegate.getResource().getPath());
-                if (lastModifiedTime.equals(_lastModifiedValue))
+                if (!_delegate.getResource().exists())
+                    return false;
```

Comment:
Aren't these redundant checks now that the constructor checks them?

Reply:
Done